### DISCUSSION TITLE
Rectangle Corners

### DISCUSSION THREAD

**Discussion by Polsulpicien** [python-pillow/Pillow#6192] [Q&A] [answered]
Hello! I was wondering if anyone knew how to do the same rectangle corners 
![image](https://user-images.githubusercontent.com/70817001/162611173-0e10a469-5fa0-44bf-9292-4b4d43d5fcdd.png)

Thank you :)

**Comment by radarhere** [comment, marked as answer]
Are you looking for https://pillow.readthedocs.io/en/stable/reference/ImageDraw.html#PIL.ImageDraw.ImageDraw.rounded_rectangle ?

```python
from PIL import Image, ImageDraw
im = Image.new("RGB", (20, 60))
draw = ImageDraw.Draw(im)
draw.rounded_rectangle((-10, 10, 10, 50), 3, fill="#999")
im.save("rounded_rectangle.png")
```
![rounded_rectangle](https://user-images.githubusercontent.com/3112309/162615711-607bb829-05aa-4a27-bd84-e40f554c7d9a.png)

**Reply by Polsulpicien** [reply]
Hey! Thank you for your answer, but if you look on the image I sent it's not rounded this is why I am asking :)

**Reply by radarhere** [reply]
Here's an adjusted version to more closely match your image.
```python
from PIL import Image, ImageDraw
im = Image.new("RGB", (15, 30))
draw = ImageDraw.Draw(im)
draw.rounded_rectangle((-10, 5, 10, 25), 1, fill="#999")
im.save("rounded_rectangle.png")
```
![rounded_rectangle](https://user-images.githubusercontent.com/3112309/162642011-d729fc0e-d269-4410-9906-917ad3fc27a3.png)

**Reply by radarhere** [reply]
Let us know if this still hasn't answered your question. In my estimation, the rectangle you have shown is rounded, albeit only slightly.

If you're not asking for a rounded rectangle, you could draw a rectangle and then a line next to it.
